poppy/wfe.py

```diff
@@ -353,6 +353,231 @@ def get_opd(self, wave):
+    :math:`\beta`: astropy quantity
+        The normalization constant. In units of :math: `\frac{m^{2}}{m^{\alpha-2}}`
+        Numerator assumes surface units of meters
+        Denominator assumes spatial frequency units are 1/m
+    :math:`L_{0}`: astropy quantity
+        The outer scale value, where the low spatial frequency flattens. Units: m
+    :math:`l_{0}`: float
+        Inner scale value, where the high spatial frequency flattens.
+    :math:`\beta_{sr}`: astropy quantity
+        Surface roughness normalization. Should match units of PSD.
+    
+    References:
+    Males, Jared. MagAO-X Preliminary-Design Review, 
+        Section 5.1: Optics Specifications, Eqn 1
+        https://magao-x.org/docs/handbook/appendices/pdr/
+    Lumbres, et al. In Prep.
 
-        return opd
+    Parameters
+    ----------
+    name : string
+        name of the optic
+    psd_parameters: list of lists
+        List of specified PSD parameters.
+        If there are multiple PSDs, then each list element is a list of specified PSD parameters.
+        i.e. [ [PSD_list_0], [PSD_list_1]]
+        The PSD parameters in a list are ordered as follows:
+        [alpha, beta, outer_scale, inner_scale, surf_roughness]
+        where:            
+            alpha: float 
+                The PSD index value.
+            beta: astropy quantity
+                The normalization constant. In units of :math: `\frac{m^{2}}{m^{\alpha-2}}`
+                Numerator assumes surface units of meters
+                Denominator assumes spatial frequency units are 1/m
+            outer_scale: astropy quantity
+                The outer scale value, where the low spatial frequency flattens. 
+                Unit requirement: meters
+            inner_scale: float
+                Inner scale value, where the high spatial frequency flattens.
+            surf_roughness: astropy quantity
+                Surface roughness normalization. Should match units of PSD.
+    psd_weight: iterable list of floats
+        Specifies the weight muliplier to set onto each model PSD
+    seed : integer
+        Seed for the random phase screen generator
+    apply_reflection: boolean
+        Applies 2x scale for the OPD as needed for reflection.
+        Default to False. 
+        Set to True if the PSD model only accounts for surface.
+    screen_size: integer
+        Sets how large the PSD matrix will be calculated.
+        The PSD matrix needs to be larger than the wavefront for Fourier transform padding purposes.
+        If None passed in, then code will default size to 4x wavefront's side.
+        Default to None.
+    rms: astropy quantity
+        Optional. Use this to force the wfe RMS
+        If a value is passed in, this is the surface rms value (not OPD) in meters.
+        If None passed, then the wfe RMS produced is what shows up in PSD calculation.
+        Default to None.
+    incident_angle: astropy quantity
+        Adjusts the WFE based on 
```

Comment:
Doc sentence cuts off halfway.

Reply:
Thanks for pointing it out. I finished the sentence. (I shouldn't be allowed to write docs at 11 PM)